site/homepage/content/release-notes.md

```diff
@@ -21,6 +21,9 @@ title = "Release Notes"
   for details regarding these new variables.
 * The *message limit* checks is now extended to include command and control messages.
   Please refer to the [resource limits] ({{% doclink "/concepts/resource-limits/" %}}) for details.
+* The health check endpoint will bind to the localhost interface and use a default 
```

Comment:
hmmm, this doesn't make much sense, does it? The idea behind using a default port was that people would get away with not specifying anything, right? However, in order to get a usable health check server, you still need to set the bind address explicitly because otherwise the server is bound to the loopback device only, i.e. it can't be accessed from any other node, which kind of makes the whole thing useless ...

That is also the reason why we originally had the sentence in the docs, that you need to set the bind address in order to get the server started. So, for the health check server we would probably want to require users to explicitly configure the bind address because otherwise the health check server is useless anyway. However, we should not by default simply start the server on `0.0.0.0` because that makes it available to all other nodes and I guess we should leave it for the user to decide if that is really intended.
WDYT?

Reply:
Yeah, that make sense. I'm so used to develop locally I forgot that `127.0.0.1` was not reachable from any other nodes :sweat_smile: 
So if I summarize the ways to get a health Check server started by doing any of the following:
For the insecure server : 
- set InsecurePortEnabled to true AND set a InsecureBindAddress  to a value other than _127.0.0.1_ .
- set InsecurePort to a value other than Constants.PORT_UNCONFIGURED AND set a InsecureBindAddress  to a value other than _127.0.0.1_ .

For the secure server : 
- set a BindAddress  to a value other than _127.0.0.1_ AND provide valid key material.
- set Port to a value other than Constants.PORT_UNCONFIGURED AND set a BindAddress  to a value other than _127.0.0.1_  AND provide valid key material.

If that sounds correct I'll implement that

Reply:
That sounds good :-) Make sure to keep the check for making sure that secure and insecure port are different, though ...